docs/source/user_guide/automl.ipynb

```diff
@@ -994,6 +994,11 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.8.6"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "425958b484b95e5a5ab632cc76b33302e47efa72ef6819296d6f51644ebb328b"
```

Comment:
I don't recall adding any of this, I wonder where it came from?

Reply:
did you open the notebook using vscode? Not sure if `make lint-fix` covers this part of the notebook but you could always manually remove it by opening the notebook in a text editor.

Reply:
I did open the notebook using vscode. I'll remove these from now on.